### PR TITLE
Use newer net-ssh as default requirement

### PR DESCRIPTION
**Pull request by Fauli83**
For some devices oxidized has problems to connect using ssh with the current Version. If you install a newer Version of net-ssh (4.1.0) The problem is solved.

An Example is the following issue:
Trying to create new model Audiocodes Mediant #758 

I needed to modify ssh.rb because in newer versions of net-ssh it seems you cannot pass an nil value for Proxy.

I also added a new model AudioCodes, which will only work with the changes above.

### CODE REVIEW

File: lib/oxidized/input/ssh.rb [hunk 1]
```diff
@@ -24,20 +24,23 @@ def connect node
...
+                :auth_methods => %w(none publickey password keyboard-interactive),
+                :number_of_password_prompts => 0,
+        }
+
       if proxy_host = vars(:ssh_proxy)
         proxy_command =  "ssh "
         proxy_command += "-o StrictHostKeyChecking=no " unless secure
         proxy_command += "#{proxy_host} -W %h:%p"
         proxy =  Net::SSH::Proxy::Command.new(proxy_command)
+        ssh_opts[:proxy] = proxy
```
Comment (ytti) [inline lib/oxidized/input/ssh.rb]: Why change in same commit the way ssh proxy is done?

File: lib/oxidized/model/audiocodes.rb [hunk 2]
```diff
@@ -0,0 +1,32 @@
+class AudioCodes < Oxidized::Model
```
Comment (ytti) [inline lib/oxidized/model/audiocodes.rb]: This belongs in another commit.

File: lib/oxidized/input/ssh.rb (hunk 1 above)
Comment (Fauli83) [inline lib/oxidized/input/ssh.rb]: You can't pass an empty proxy in ssh_opts or you'll receive errors in the log. That's why I've changed it.

File: lib/oxidized/model/audiocodes.rb (hunk 2 above)
Comment (Fauli83) [inline lib/oxidized/model/audiocodes.rb]: you're right. removed it and created new request

**Comment by ytti** [comment]
Looks good. Can you also squash these commits so this does not bring any audicodecs addition and deletion to the history. 

Thanks!

**Comment by Fauli83** [comment]
New request with cleaner commits .> #909
